src/guest_memory.rs

```diff
@@ -563,6 +579,18 @@ pub trait GuestMemory {
             .ok_or_else(|| Error::InvalidGuestAddress(addr))
             .and_then(|(r, addr)| r.get_host_address(addr))
     }
+
+    /// Returns a [`VolatileSlice`](struct.VolatileSlice.html) of `count` bytes starting at
+    /// `addr`.
+    fn get_slice(
+        &self,
+        addr: GuestAddress,
+        count: usize,
+    ) -> Result<volatile_memory::VolatileSlice> {
+        self.find_region(addr)
+            .ok_or_else(|| Error::InvalidGuestAddress(addr))
+            .and_then(|r| r.get_slice(r.to_region_addr(addr).unwrap(), count))
```

Comment:
Please add a comment to specify why is this unwrap safe.

Reply:
You  can actually use `GuestMemory::to_region_addr`, like

```rust
    self.to_region_addr(addr)
        .ok_or_else(|| Error::InvalidGuestAddress(addr))
        .and_then(|(r, addr)| r.get_slice(addr, count))
```